### PR TITLE
obsolete MONDO:0009148

### LINKED ISSUE #432: merge MONDO_0009148 'Rosselli-Gulienetti syndrome' with MONDO_0007124 'ankyloblepharon-ectodermal defects-cleft lip/palate syndrome'
MONDO_0009148 'Rosselli-Gulienetti syndrome'

From comments in ontology:
obsolete in Orphanet. Wikipedia suggests a merge: it has been suggested that this syndrome, AEC syndrome (MONDO_0007124 'ankyloblepharon-ectodermal defects-cleft lip/palate syndrome') and Rapp–Hodgkin syndrome may be variations of the same disease

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
closes #432

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -126416,19 +126417,25 @@ synonym: "ankyloblepharon-ectodermal defects-cleft LIP/palate" RELATED [OMIM:106
...
 synonym: "Seres-Santamaria Arimany Muniz syndrome" RELATED [GARD:0004805]
 xref: DOID:0090119 {source="MONDO:equivalentTo"}
 xref: GARD:0004805 {source="MONDO:equivalentTo"}
 xref: GARD:0006571 {source="MONDO:equivalentTo", source="Orphanet-shared"}
 xref: ICD10:Q82.4 {source="ORDO:1071/attributed", source="ORDO:1071/ntbt", source="Orphanet:1071", source="DOID:0090119"}
 xref: MESH:C535289 {source="MONDO:superClassOf", source="Orphanet:1071", source="ORDO:1071/e", source="DOID:0090119"}
 xref: MESH:C535847 {source="MONDO:equivalentTo", source="MONDO:ontobio"}
+xref: MESH:C563117 {source="MONDO:equivalentTo", source="MONDO:ontobio"}
 xref: OMIM:106260 {source="MONDO:equivalentTo", source="Orphanet:1071", source="GARD:0004805", source="ORDO:1071/e", source="DOID:0090119"}
+xref: OMIM:225000 {source="MONDO:equivalentTo"}
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo]: proxy merge

**Comment by cmungall** [comment]
still a proxy merge. Intentional?

**Comment by nicolevasilevsky** [comment]
@cmungall In this case, looks like Orphanet had obsoleted this class (see Annie's comment on the ticket: #432), but OMIM still has this as two separate terms.

Who do we want to follow? Do we want to follow OMIM and keep them separate or follow Orphanet and merge them into one term?

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -171331,17 +171338,10 @@ property_value: confidence "1.013902333975493" xsd:double
 
 [Term]
 id: MONDO:0009148
-name: Rosselli-Gulienetti syndrome
-def: "A rare congenital ectodermal dysplasia syndrome with a range of signs and symptoms including cleft lip or palate, mental retardation and various forms of ectodermal dysplasia. Additional symptoms may include fused eyelids, absent nails, delayed bone growth and dry skin. It is believed that this syndrome follows an autosomal dominant pattern of inheritance with incomplete penetrance, and caused by a mutation affecting the TP63 gene" [https://en.wikipedia.org/wiki/Rosselli%E2%80%93Gulienetti_syndrome]
+name: obsolete Rosselli-Gulienetti syndrome
 comment: obsolete in Orphanet. Wikipedia suggests a merge: it has been suggested that this syndrome, AEC syndrome and Rapp–Hodgkin syndrome may be variations of the same disease
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo]: I may have written the original comment. In cases like this where we have disagreement I would state this in terms of the clinical/medical reasons for merging, I would not note wikipedita

**Comment by nicolevasilevsky** [comment]
I followed up again

**Comment by nicolevasilevsky** [comment]
Ada said OMIM will not merge these terms. I'll close this PR.

**Pull request closed without merging**